boms/geode-all-bom/src/test/resources/expected-pom.xml

```diff
@@ -877,6 +877,12 @@
         <version>2.4.1</version>
         <scope>compile</scope>
       </dependency>
+      <dependency>
+        <groupId>org.jetbrains</groupId>
+        <artifactId>annotations</artifactId>
+        <version>20.1.0</version>
+        <scope>compile</scope>
```

Comment:
The seems off. I would expect the default scope for this type to be `compileOnly`, though I don't think it really matters much in a bom.

Reply:
is there any such thing as a `compileOnly` scope for a Maven project?

Reply:
sort of... compile scoped with optional true.